.github/workflows/runnable_cxx.yml

```diff
@@ -56,17 +57,13 @@ jobs:
       # very few PRs actually benefit from this.
       fail-fast: false
       matrix:
-        # Mac OS 11, Ubuntu 18.04 in order to support older LLVM / GCC
-        # Ubuntu 20.04 in order to support newer LLVM / GCC
-        os: [ macOS-11, ubuntu-18.04, ubuntu-20.04, windows-2019 ]
+        os: [ macOS-11, ubuntu-20.04, windows-2019 ]
 
         target: [
-          # Versions of clang earlier than 8 are not available on 18.04
-          # Versions of clang earlier than 11 are not available on 20.04
+          # Versions of clang earlier than 11 are not available on 20.04, but are on macOS-11
           clang-13.0.0, clang-12.0.0, clang-11.0.0, clang-10.0.0, clang-9.0.0, clang-8.0.0,
-          # For g++, we test all major's latest minor releases since the
-          # introduction of ABI the dual ABI (v5.1, 2015-04-15)
-          g++-11, g++-10, g++-9, g++-8, g++-7, g++-6, g++-5,
+        # For g++, we test the oldest compiler on Ubuntu 20.04, which is GCC-9
+          g++-11, g++-10, g++-9,
```

Comment:
No g++-12?

Reply:
I'll look into adding support in another PR